README.md

```diff
@@ -246,6 +246,28 @@ AWX:
 
 ## Values Summary
 
+### Controller
+The configuration of the `awx-operator-controller-manager` `Deployment` resource can be overridden by the
+`operator-controller` field. Any fields specified under this key will map directly onto the root hierarchy of the [Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) configuration.
+
+For example, to override the replicas of the controller deployment, use:
```

Comment:
While the example of the replica is valid. This is not really a common scenario to customize the operator replica (maybe to scale up/scale down). One more common use case is to specify a different container image for each container as seen in [this issue](https://github.com/ansible-community/awx-operator-helm/issues/17) or [that one](https://github.com/ansible-community/awx-operator-helm/issues/27) to cover disconnected/restricted environments.

What do you think? Could you maybe provide an example on how to do that @miles-w-3 

Thanks for the PR btw!

Reply:
@schen1 Perhaps we should capture these items in a follow up to improve the docs. While I think extensive documentation is always a good thing, I'm not sure we should hold up this PR much longer. Any thoughts @miles-w-3 ?

Reply:
fine for me :)

Reply:
I am fine either way. The use case for overriding an image tag is tricky because of the way lists are merged in helm. 

Lists are overridden instead of merged for each layer. So under this proposal, if you want to change the image version, you have to override the entire container list spec. For that reason, this solution is more optimal for adding things like annotations or service account specs to the controller's pods. 

If there is desire for more targeted overrides of specific containers, that could be another feature for a template that iterates through a list, matches a container name, and then merges a set override

Reply:
Thanks for the details and the great contribution. Then it makes sense to have it as a different feature/PR